### ISSUE TITLE
B008: false positive with lambda

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8-bugbear#255]
```python
def f(x=lambda y: y(1)):
    pass
```

expected: no errors

got:

```console
$ flake8 --version
4.0.1 (flake8-bugbear: 22.3.23, mccabe: 0.6.1, pycodestyle: 2.8.0, pyflakes: 2.4.0)
CPython 3.8.10 on Linux
$ flake8 t.py 
t.py:1:19: B008 Do not perform function calls in argument defaults.  The call is performed only once at function definition time. All calls to your function will reuse the result of that definition-time function call.  If this is intended, assign the function call to a module-level variable and use that variable as a default value.
```

**Comment by asottile** [comment]
seems already fixed on master, nevermind!
